src/main/java/io/strimzi/kafka/bridge/KafkaBridgeConsumer.java

```diff
@@ -93,8 +93,12 @@ public void create(Properties config, String groupId) {
      * @param topicSubscriptions topics to subscribe to
      */
     public void subscribe(List<SinkTopicSubscription> topicSubscriptions) {
+        if (topicSubscriptions == null) {
+            throw new IllegalArgumentException("Topic subscriptions cannot be null");
+        }
+
         if (topicSubscriptions.isEmpty()) {
-            throw new IllegalArgumentException("At least one topic to subscribe has to be specified!");
+            this.unsubscribe();
```

Comment:
so after unsubscribing you are actually not returning and the subscribe operation will run anyway. It should not cause a problem just because the Kafka subscribe with empty topic list will behave as an unsubscribe.
For this reason I would say that this addition about calling unsubscribe is not needed, we should just allow an empty topics list and then allowing the code to run the subscribe.